Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+RUN yum -y install https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
```

Comment:
I'd move those yum into the big long series of yum commands so that the cache from this command doesn't stick around. Likely won't be a large cache but all enabled repos metadata will get cached which can be like 50MB maybe, so not huge deal.